### PR TITLE
add native installation for rockylinux

### PR DESCRIPTION
**Pull request by maneamarius**
##### Summary
Add native installation support for Rocky Linux

##### Test Plan
Run kickstart.sh in Rocky Linux and make sure it was installed via rpm package from netdata-repo-edge

### CODE REVIEW

File: packaging/installer/kickstart.sh [hunk 1]
```diff
@@ -877,6 +878,24 @@ try_package_install() {
...
+    rocky)
+      if command -v dnf > /dev/null; then
+        pm_cmd="dnf"
+        repo_subcmd="makecache"
+      else
+        pm_cmd="yum"
+      fi
+      SYSVERSION_ROCKY="8"
+      #repo_prefix="el/${SYSVERSION}"
+      repo_prefix="el/${SYSVERSION_ROCKY}"
```
Comment (ilyam8) [inline packaging/installer/kickstart.sh:890]: - do we need the commented-out line?
- do we need `SYSVERSION_ROCKY` as a variable? My understanding is that is manually set and always 8.

File: packaging/installer/kickstart.sh [hunk 2]
```diff
@@ -421,6 +421,7 @@ get_system_info() {
             ;;
           rhel)
             DISTRO_COMPAT_NAME="centos"
+            DISTRO_COMPAT_NAME="rocky"
```
Comment (ilyam8) [inline packaging/installer/kickstart.sh:424]: Here you reassign the `DISTRO_COMPAT_NAME` var. is this intentional?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -17,7 +17,7 @@ on:
         default: nightly
         required: true
 concurrency: # This keeps multiple instances of the job from running concurrently for the same ref and event type.
-  group: release-${{ github.ref }}-${{ github.event_name }}
+  group: build-${{ github.ref }}-${{ github.event_name }}
```
Comment (Ferroin) [inline .github/workflows/build.yml:20]: This is actually something we _should_ do, but it should be in a separate PR.

File: packaging/installer/kickstart.sh [hunk 4]
```diff
@@ -877,6 +879,22 @@ try_package_install() {
...
+      fi
+      repo_prefix="el/8"
+      pkg_type="rpm"
+      pkg_suffix=".noarch"
+      pkg_vsep="-"
+      pkg_install_opts="${interactive_opts}"
+      repo_update_opts="${interactive_opts}"
+      uninstall_subcmd="remove"
+      INSTALL_TYPE="binpkg-rpm"
+      ;;
```
Comment (Ferroin) [inline packaging/installer/kickstart.sh:897]: If we’re setting `DISTRO_COMPAT_NAME` correctly, this case should not be needed (IOW, the variables should be identical between Rocky and CentOS, so we set `DISTRO_COMPAT_NAME` to `centos` and just use that case in this case statement).

**Review by Ferroin — CHANGES_REQUESTED** [review]

File: packaging/installer/kickstart.sh [hunk 5]
```diff
@@ -421,6 +421,8 @@ get_system_info() {
             ;;
           rhel)
             DISTRO_COMPAT_NAME="centos"
+          rocky)
```
Comment (Ferroin) [inline packaging/installer/kickstart.sh:424]: This should just be merged with the above case, using `rocky|rhel` as the matching expression.

File: packaging/installer/kickstart.sh (hunk 4 above)
Comment (ilyam8) [reply packaging/installer/kickstart.sh]: The only difference is `repo_prefix="el/${SYSVERSION}"` part (assuming it is always 8 for rocky, could be 7 for centos).

Comment (maneamarius) [reply packaging/installer/kickstart.sh]: for rockylinux we need to use `el/8` otherwise it won't work

**Review by iigorkarpov — DISMISSED** [review]

File: packaging/installer/kickstart.sh (hunk 5 above)
Comment (ilyam8) [reply packaging/installer/kickstart.sh:425]: @maneamarius that is not fixed

File: packaging/installer/kickstart.sh [hunk 6]
```diff
@@ -868,6 +871,7 @@ try_package_install() {
       else
         pm_cmd="yum"
       fi
+      SYSVERSION=$(echo "$SYSVERSION" | cut -d'.' -f1)
```
Comment (Ferroin) [inline packaging/installer/kickstart.sh:874]: This should probably go up in the `get_system_info()` function instead of here.

Comment (maneamarius) [reply packaging/installer/kickstart.sh:874]: we cannot remove the stuff after the dot for all operating systems, as Opensuse has 15.2, 15.3 versions and need to stay like that..

Comment (Ferroin) [reply packaging/installer/kickstart.sh:871]: I don’t mean do it for all platforms, just have the special case for it up in `get_system_info()` instead of in the code that attempts to install a native package.

`try_package_install()` is not guaranteed to be called,  and the `SYSVERSION` variable is one of the things that gets reported by telemetry, so there’s a risk of handling this here resulting in inconsistencies in our anonymous statistics. By mangling the version in `get_system_info()` instead, we avoid that possibility (and also ensure that every other part of the script that may care can count on the value being consistent no matter what execution path is taken).

Comment (maneamarius) [reply packaging/installer/kickstart.sh:871]: agreed. I will make this change now

**Review by Ferroin — APPROVED** [review]

**Review by ilyam8 — APPROVED** [review]
